### ISSUE TITLE
Multi Datacenter Repair

### ISSUE THREAD

**Issue by shaurya10000** [BrianGallew/cassandra_range_repair#62]
Hi

https://github.com/BrianGallew/cassandra_range_repair#multiple-datacenters
mentions-

> "If you have multiple datacenters in your ring, then you MUST specify the name of the datacenter containing the node you are repairing as part of the command-line options (--datacenter=DCNAME). Failure to do so will result in only a subset of your data being repaired (approximately data/number-of-datacenters). This is because nodetool has no way to determine the relevant DC on its own, which in turn means it will use the tokens from every ring member in every datacenter."

So, if we are running repair on every node in the multi DC Cassandra cluster then do we need to specify --datacenter=name on every node or are we good without specifying the datacenter ?

**Comment by BrianGallew** [comment]
Nope.  Each node, in each DC, will end up doing a subset of what it owns
rather than all of what it owns.

**Comment by shaurya10000** [comment]
Hi Brian
Thanks for your reply.
It's 'No' for providing the data center option or for not providing it ?
Sorry for being too picky, on basis of your answer I'll have to inform the team for correct parameter.

Although I went through the code and looks like it is must to specify the data center else code sorts and creates wrong token ranges.

**Comment by BrianGallew** [comment]
To be clear: you MUST specify the DC in any multi-DC deployment just like
the docs say.
